_posts/2023-01-25-snowflake-snowpro-core-cert.md

```diff
@@ -1,18 +1,20 @@
 ---
 layout: postv2
-hidden: true
 inject_optimize_tag: true
 canonical_url: https://mechanicalrock.github.io/2023/01/25/snowflake-snowpro-core-cert.html
 font: serif
 title: "Snowflake SnowPro Core Certification notes"
-description: ""
+description: "Snowflake SnowPro Core Certification study guide"
 date: 2023-01-25
 highlight: monokai
-image: /img/skeleton-loaders/banner.jpg
```

Comment:
Don't you need this one as well?  I thought this one was included in meta-tags when used in LI posts, website, etc

Reply:
It was duplicating the image in localhost when I used it